src/Api/Response/Parser/FiltersParser.php

```diff
@@ -107,6 +108,16 @@ public function parseFilterItem($filterType, &$filterItem, $data, $index)
                 $filterItem['priceMax'] = $data->parameters->max;
             }
 
+            if ($filterType === Plugin::FILTER_TYPE_IMAGE) {
+                if (isset($data->image) && $data->image->__toString() !== '' && $data->image->__toString()[0] !== '/') {
+                    $filterItem['imageUrl'] = null;
+
+                    if ($this->colorImageService->isImageAccessible($data->image->__toString())) {
```

Comment:
Since it is not only about colors anymore:
* Rename `$this->colorImageService` to `$this->imageService`.
* Also rename the class itself accordingly.